### PR TITLE
docs(protocol): update mainnet deployment log

### PR DESCRIPTION
**Pull request by davidtaikocha**

### CODE REVIEW

File: packages/protocol/deployments/mainnet-contract-logs.md [hunk 1]
```diff
@@ -150,25 +152,26 @@
...
-- impl: `0x9fBBedBBcBb753E7214BE08381efE10d89D712fE`
+- impl: `0xe0A5D394878723CEAEC8B993e04756DF1f4B44eF`
 - owner: `admin.taiko.eth`
 - logs:
   - deployed on May 1, 2024 @commit `56dddf2b6`
   - Upgraded from `0x99Ba70E62cab0cB983e66F72330fBDDC11d85501` to `0x9fBBedBBcBb753E7214BE08381efE10d89D712fE` @commit `b90b932` @tx`0x416560cd96dc75ccffebe889e8d1ab3e08b33f814dc4a2bf7c6f9555071d1f6f`
   - `Init2()` called and reset block hash to `0411D9F84A525864E0A7E8BB51667D49C6BF73820AF9E4BC76EA66ADB6BE8903` @tx `0x7311fee56f87294e336393b55939489bc1e810c402f304013475d04c90ca32a9`
-- todo:
-  - upgrade the implementation again
+  - Upgraded from `0x9fBBedBBcBb753E7214BE08381efE10d89D712fE` to `0xe0A5D394878723CEAEC8B993e04756DF1f4B44eF` on May 21 @commit `c817e76d9` @tx``
```
Comment (dantaik) [inline packages/protocol/deployments/mainnet-contract-logs.md:174]:
```suggestion
  - Upgraded from `0x9fBBedBBcBb753E7214BE08381efE10d89D712fE` to `0xe0A5D394878723CEAEC8B993e04756DF1f4B44eF` on May 21 @commit `c817e76d9` @tx`0xaed098ad0c93113e401f61358f963501f40a046c5b5b659a1610f10120a9a86b`
```

File: packages/protocol/deployments/mainnet-contract-logs.md [hunk 2]
```diff
@@ -150,25 +152,26 @@
...
   - proposer: `0x000000633b68f5d8d3a86593ebb815b4663bcbe0`
+  - chain_watchdog: `0xE3D777143Ea25A6E031d1e921F396750885f43aC`
 - logs:
   - deployed on May 1, 2024 @commit `56dddf2b6`
   - admin.taiko.eth accepted the ownership @tx`0x0ed114fee6de4e3e2206cea44e6632ec0c4588f73648d98d8df5dc0183b07885`
   - Upgraded from `0xd912aB787624c9eb96a37e658e9596e114360440` to `0xF1cA1F1A068468E1dcF90dA6add185467de80943` @commit `b90b932` @tx`0x416560cd96dc75ccffebe889e8d1ab3e08b33f814dc4a2bf7c6f9555071d1f6f`
   - `Init2()` called @tx `0x7311fee56f87294e336393b55939489bc1e810c402f304013475d04c90ca32a9`
 - todo:
   - register `prover_set`
+  - register `chain_watchdog` on May 21 @tx ``
```
Comment (dantaik) [inline packages/protocol/deployments/mainnet-contract-logs.md:163]:
```suggestion
  - register `chain_watchdog` on May 21 @tx `0xaed098ad0c93113e401f61358f963501f40a046c5b5b659a1610f10120a9a86b`
```

Comment (dantaik) [reply packages/protocol/deployments/mainnet-contract-logs.md:163]: we should remove `chain_watchdog` registration until we tested the feature in hekla.

We can do it with the Bridge upgrade.

Comment (davidtaikocha) [reply packages/protocol/deployments/mainnet-contract-logs.md]: added a todo

**Review by dantaik — APPROVED** [review]

**Comment by dantaik** [comment]
Lets get rid of all TODOs before merging this PR

**Review by smtmfft — APPROVED** [review]

**Review by xiaodino — APPROVED** [review]

**Review by YoGhurt111 — APPROVED** [review]

**Pull request merged**